guides/customizing-component-styles.md

```diff
@@ -0,0 +1,70 @@
+# Customizing Angular Material component styles
+
+### Styling concepts
+
+There are 3 questions to keep in mind while customizing the styles of Angular Material
+components:
+
+1. Are your styles encapsulated?
+2. Are your styles more specific than the defaults?
+3. Is the component a child of your component, or does it exist elsewhere in the DOM?
+
+##### View encapsulation
+
+By default, Angular component styles are scoped to affect the component's view. This means that
+the styles you write will affect all the elements in your component template. They will *not*
+affect elements that are children of other components within your template. You can read more
+about view encapsulation in the
+[Angular documentation](https://angular.io/guide/component-styles#view-encapsulation).
```

Comment:
Would be good to include a link to the recent blog post on this subject as well: https://blog.angular.io/the-state-of-css-in-angular-4a52d4bd2700
(which was partially ghostwritten by me)